### PR TITLE
[hotfix][python] Aligns with Java Table API by removing methods exec_env and query_config

### PR DESCRIPTION
**Pull request by dianfu**
## What is the purpose of the change

*This pull request aligns Python Table API with the Java Table API by removing methods exec_env() and query_config() in table_environment.py*

## Brief change log

  - *Removes methods exec_env() and query_config() in table_environment.py*

## Verifying this change

This change is a trivial rework without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: flink-python/pyflink/shell.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
     *         os.remove(sink_path)
     *     else:
     *         shutil.rmtree(sink_path)
-    * bt_env.exec_env().set_parallelism(1)
+    * b_env.set_parallelism(1)
```
Comment (WeiZhong94) [inline flink-python/pyflink/shell.py]: I think we can add "b_env" at the beginning of the example to tell the users what variable is predefined. "Use the 'bt_env' variable" -> "Use the 'b_env', 'bt_env' variable"

**Comment by dianfu** [comment]
@WeiZhong94 Thanks a lot for the review. Updated.

**Comment by WeiZhong94** [comment]
LGTM, +1

**Comment by sunjincheng121** [comment]
Hi, I think we also need to update the `shell.py`, What do you think?

**Comment by dianfu** [comment]
@sunjincheng121 Thanks a lot for the review. Updated the PR.

**Comment by sunjincheng121** [comment]
Thanks for the update @dianfu 
+1 to merged.
@flinkbot approve all

**Pull request closed without merging**